### PR TITLE
[keystone] deploy seeds if it is 1st install; fix seed naming; allow to

### PR DESCRIPTION
**Pull request by galkindmitrii**
specify both public/internal external service IPs; bump chart.

### CODE REVIEW

File: openstack/keystone/templates/seed.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
-{{- if .Release.IsUpgrade }}
```
Comment (Carthaca) [inline openstack/keystone/templates/seed.yaml:1]: In an 'ordinary' keystone installation the seeds won't work on install - but it's okay I guess, the seeder will re-try to apply the seeds.

**Review by Carthaca — APPROVED** [review]
lgtm

File: openstack/keystone/templates/seed.yaml (hunk 1 above)
Comment (galkindmitrii) [reply openstack/keystone/templates/seed.yaml:1]: yes, correct. The problem is that now we have to run upgrade / 2nd deployment before the seeds even appear. Looking at other services, almost nobody seem to have this clause right now.

**Comment by galkindmitrii** [comment]
Merci

**Pull request merged**